packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -1150,48 +1158,52 @@ class _DropdownMenuState<T> extends State<DropdownMenu<T>> {
           child: widget.leadingIcon ?? const SizedBox.shrink(),
         );
 
-        final Widget textField = TextField(
-          key: _anchorKey,
-          enabled: widget.enabled,
-          mouseCursor: effectiveMouseCursor,
-          focusNode: widget.focusNode,
-          canRequestFocus: canRequestFocus(),
-          enableInteractiveSelection: canRequestFocus(),
-          readOnly: !canRequestFocus(),
-          keyboardType: widget.keyboardType,
-          textAlign: widget.textAlign,
-          textAlignVertical: TextAlignVertical.center,
-          maxLines: widget.maxLines,
-          textInputAction: widget.textInputAction,
-          cursorHeight: widget.cursorHeight,
-          style: effectiveTextStyle,
-          controller: _effectiveTextEditingController,
-          onEditingComplete: _handleEditingComplete,
-          onTap: !widget.enabled
-              ? null
-              : () {
-                  handlePressed(controller, focusForKeyboard: !canRequestFocus());
-                },
-          onChanged: (String text) {
-            controller.open();
-            setState(() {
-              filteredEntries = widget.dropdownMenuEntries;
-              _enableFilter = widget.enableFilter;
-              _enableSearch = widget.enableSearch;
-            });
-          },
-          inputFormatters: widget.inputFormatters,
-          decoration: InputDecoration(
-            label: widget.label,
-            hintText: widget.hintText,
-            helperText: widget.helperText,
-            errorText: widget.errorText,
-            prefixIcon: widget.leadingIcon != null
-                ? SizedBox(key: _leadingKey, child: widget.leadingIcon)
-                : null,
-            suffixIcon: widget.showTrailingIcon ? trailingButton : null,
-          ).applyDefaults(effectiveInputDecorationTheme),
-          restorationId: widget.restorationId,
+        final Widget textField = Semantics(
+          button: !canRequestFocus(),
```

Comment:
some platform may still treat this as textfield if both istextField and isButton are true.

Reply:
Can this be focusable but not enabled?

Reply:
> some platform may still treat this as textfield if both istextField and isButton are true.

I see. I added `textField` and assigned it the opposite value of `button`.

> Can this be focusable but not enabled?

When it's disabled, the text field is not focusable.